Source/NSException.m

```diff
@@ -846,7 +875,19 @@ NSUInteger NSCountFrames(void)
 GSPrivateReturnAddresses(NSUInteger **returns)
 {
   unsigned      numReturns;
-#if HAVE_BACKTRACE
+#if defined(HAVE_UNWIND_H)
```

Comment:
Does this order lead to unwind being used instead of backtrace if both are available? And are there system where we could have both? Which one would be preferable?

Reply:
You’re right currently unwind would be used over backtrace if both are available, because unwind seemed like the more modern library but I’m not sure which one is preferable. Does anyone know? Otherwise I can just switch it so we don’t change current behavior.

Reply:
Thinking about this some more it does make sense to me to keep backtrace() as the preferred solution as it likely has better results for symbolication. I’ve updated the PR to prefer backtrace over unwind, so the change now only applies to systems without support for backtrace (e.g. Android).